drivers/cyw43/cywbt.c

```diff
@@ -81,10 +88,10 @@ STATIC int cywbt_hci_cmd_raw(size_t len, uint8_t *buf) {
 
     int sz = buf[2] - 3;
     for (int i = 0; i < sz; ++i) {
-        while (!uart_rx_any(&mp_bluetooth_hci_uart_obj)) {
+        while (!mp_bluetooth_hci_uart_any()) {
             MICROPY_EVENT_POLL_HOOK
         }
-        buf[i] = uart_rx_char(&mp_bluetooth_hci_uart_obj);
+        buf[i] = mp_bluetooth_hci_uart_readchar();
```

Comment:
Please also change this to the same pattern as above.

Reply:
Can we instead save a line and:

```c
    for (int c, i = 0; i < sz; ++i) {
        while ((c = mp_bluetooth_hci_uart_readchar()) == -1) {
            MICROPY_EVENT_POLL_HOOK
        }
        buf[i] = c;
    }
```
?